node-definitions/sonic/SONiCv.yaml

```diff
@@ -0,0 +1,61 @@
+id: SONiCv
+general:
+  nature: switch
+  read_only: false
+device:
+  interfaces:
+    has_loopback_zero: true
+    physical:
+      - Eth0
+      - Ethernet0
+      - Ethernet4
+      - Ethernet8
+      - Ethernet12
+      - Ethernet16
+      - Ethernet20
+      - Ethernet24
+      - Ethernet28
+    serial_ports: 2
```

Comment:
Which two serial ports does it have?  A console and an aux?

Reply:
Can be single console port